runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -94,12 +95,16 @@
 @PluginDescriptor(
 	name = "Timers",
 	description = "Show various timers in an infobox",
-	tags = {"combat", "items", "magic", "potions", "prayer", "overlay"}
+	tags = {"combat", "items", "magic", "potions", "prayer", "overlay", "abyssal", "sire"}
 )
 public class TimersPlugin extends Plugin
 {
+	private static final int ABYSSAL_SIRE_STUNNED_NPC_ID = NpcID.ABYSSAL_SIRE_5887;
+
 	private int lastRaidVarb;
 
+	private boolean abyssalSireStunTimerShown;
```

Comment:
This variable seems useless; it is only set within the same method that checks it--just default to removing the stun timer if the changed NPC doesn't match the stunned sire ID.